domains/transfer-agreements/TransferAgreements.API/API/ApiModels/Responses/TransferAgreementAuditDto.cs

```diff
@@ -0,0 +1,8 @@
+namespace API.ApiModels.Responses;
+
+public record TransferAgreementAuditDto(
```

Comment:
Why no start date? 
Why no information of when the change was performed?
Why no information about the type of change?